statement on FSF reappointing Stallman to the board of directors (#224)
mixxxdj/website: content/news/2021-XX-XX-fsf-statement.md

> title: "A Statement on Richard Stallman"

Reviewer: ```suggestion
title: "A Statement on Richard Stallman and the Free Software Foundation"
```

Just a suggestion, I don't mind much either way

> authors: Owen Williams

Reviewer: Maybe we should sign this as "The Mixxx Team"?

Author: yes definitely

tags: community, diversity, politics
date: 2021-03-26 00:00:00

_A Statement from Mixxx on Richard Stallman's return to the FSF._

The contributors to the Mixxx project were dismayed this week to learn that Richard Stallman had been reinstated to the board of directors of the Free Software Foundation.

> His departure in 2019 was a partial acknowledgement of the [harm he's caused over multiple decades](https://selamjie.medium.com/remove-richard-stallman-appendix-a-a7e41e784f88), including creating a sexually hostile work and learning environment, sexual harassment, and abuse of power.
> Loss of his position in the community was only the barest of consequences, but his resignation came as a relief for many in the community who have suffered his bad behavior.

Reviewer 2: ```suggestion
His departure in 2019 was a consequence of the criticism he has faced regarding his inappropriate behavior towards women, his hurtful comments about people with Down's Syndrome and his remarks regarding the Epstein scandal. 
His resignation came as a relief for many who have suffered his bad behavior.
We as free software developers strive build an inclusive and welcoming community and all of these points disqualify Richard Stallman to represent the FSF and the free software community as a whole.
```

Author: done (slightly tweaked)

While Mixxx is not officially associated with the Free Software Foundation,

> there's an inevitable connection in the public's mind between all FOSS projects and the FSF.

Reviewer: Moreover, Mixxx is and always has been licensed under the terms of the GNU General Public License, version 2 *or later*, and only the Free Software Foundation can publish new versions of the GNU GPL which could be retroactively applied to the Mixxx code.
Therefore the Mixxx project has an interest in ensuring that the Free Software Foundation has inclusive, principled, and wise leadership which can effectively lead the broad free software community to meet present and future challenges.

Author: added (slightly tweaked)

Therefore we feel an obligation to add our voices to the chorus of those who have expressed dismay at this turn of events and call for the resignation not only of Richard Stallman, but the entire FSF board of directors.

> By sanctioning his return, the board of directors has lost all credibility on the issues of diversity, equity, and inclusion.

Reviewer: Furthermore, we call upon the Free Software Foundation to revise its bylaws to institute democratic processes for choosing leadership.
We believe that the myriad of software developers who chose to trust the FSF with stewardship of licenses for the code we wrote deserve to have input over the leadership of the FSF.
We cannot trust an organization that knowingly makes very controversial decisions behind closed doors.

Author: added (slightly tweaked)

title: "A Statement on Richard Stallman and the Free Software Foundation"
authors: Owen Williams
tags: community, diversity, politics
date: 2021-03-26 00:00:00

> _A Statement from Mixxx on Richard Stallman's return to the FSF._

Reviewer: This is redundant with the title of the blog post. I think it should be removed.
![image](https://user-images.githubusercontent.com/9455094/112706509-a1fc1400-8e72-11eb-855c-3c0a869616d8.png)

The contributors to the Mixxx project were dismayed this week to learn that Richard Stallman had been reinstated to the board of directors of the Free Software Foundation.
His departure in 2019 was a consequence of the criticism he faced regarding his inappropriate behavior towards women, his hurtful comments about people with Down's Syndrome and his remarks relating to the Epstein scandal.
His resignation came as a relief for many who have suffered his bad behavior.

> Richard Stallman's actions disqualify him from representing the FSF and the free software community as a whole.

Reviewer 3: ```suggestion
```
We can't prove this statement and it should be clear anyway from the sentences before.

Reviewer: Whether someone is qualified for something is an opinion, not a fact. We can voice our opinion here.

Reviewer 3: Yes, but we have decided to do a blog post and not sign as org, to take the chance to explain the situation regarding to Mixxx like other orgs did.

It does not work well to express an opinion when you speak for a whole group.

Reviewer: I don't follow your reasoning. You seem to imply that you agree with the sentence but we shouldn't publish it unless we all agree? But don't we all agree on this?

Author: It's redundant to say "we believe" in a blog post about our beliefs. I can add it back but it's unnecessary verbiage because the existence of the blog post implies this is our belief as a group.

Reviewer 2: It's redundant but it doesn't hurt either. If it helps so that we can agree on this, let's just add it.

While Mixxx is not officially associated with the Free Software Foundation, there's an inevitable connection in the public's mind between all FOSS projects and the FSF.
Furthermore, Mixxx is and always has been licensed under the terms of the GNU General Public License, version 2 or *later*, and only the Free Software Foundation can publish new versions of the GNU GPL which could be retroactively applied to the Mixxx code.

> Given this relationship, the Mixxx project has an interest in ensuring that the Free Software Foundation has inclusive, principled, and wise leadership which can effectively lead the broad free software community to meet present and future challenges.

Reviewer: add paragraph break after this line

> Therefore, we feel an obligation to add our voices to the chorus of those who have expressed dismay at this turn of events and call for the resignation not only of Richard Stallman, but the entire FSF board of directors.

Reviewer 3: I think this blog post should be written in the sense of distancing from the FSF and not for calling resignation. For my feeling this was consense among the team. Else we could have sign the open letter. 
```suggestion
```

Reviewer: I think it would be negligent to not make calls for specific actions that the FSF could be held accountable for.

Reviewer 3: Or blog ist mit the place to communicate calls to the FSF. 
In addition it is pretentious to call such things from things based on hearsay.
I don't want to or part of a public storm, fueled by comments on social media.

This public rush it a strong evidence that the reinstallation of Stallman is read as movement away from inclusion.
Our business is to distancing us from that. 

Calls to the FSF need to be addressed to them.

Reviewer: > Calls to the FSF need to be addressed to them.

Private communications are trivial for them to ignore. They need to be held accountable publicly or nothing will change.

Reviewer 3: No other org has repeated this call on their webside.

Reviewer 2: I have no idea how the FSF's decision to reinstate RMS came to be. I think their decision is wrong, but demanding the whole board of directors to step down - including the female board members who had to put up with Stallman's behavior - is something I don't necessarily agree with.

Reviewer: > including the female board members who had to put up with Stallman's behavior

That's beside the point. She already stepped down.

Reviewer: Would you agree to calling for Stallman's resignation alone?

Reviewer 3: So you think the board members will read our call on the Mixxx webside and will act on it? 

The only thing this will happen if they actually read it is, they will become angry and will not treat us a serious partner. 

I think the FSF has already proposed future steps l, so this call is outdated any way.

Reviewer: > I think the FSF has already proposed future steps l, so this call is outdated any way.

They have not committed to removing all the current board of directors.

Reviewer: Doing nothing would be admitting defeat before even trying.

Reviewer 2: > > I think the FSF has already proposed future steps l, so this call is outdated any way.
> 
> They have not committed to removing all the current board of directors.

Who is "she"? Both Kat Walsh and Odile Bénassy are listed as board members on the FSF's website.

https://www.fsf.org/about/staff-and-board

Reviewer: Ah, sorry. Kat Walsh [has resigned](https://mastodon.social/@mindspillage/105948765536809476). I don't know about Odile Bénassy.

Author: By allowing Stallman to return, the board of directors is tacitly approving of his behavior and saying it's excusable. I don't see how the board can remain as well. 

What about:
"Therefore, we feel an obligation to add our voices to the chorus of those who have expressed dismay at this turn of events and call for the resignation of Richard Stallman. The FSF board of directors, by allowing him to return, is giving the appearance of endorsing his conduct and it seems doubtful that they can remain effective without significant personnel changes and bylaw reforms."

Author: (committed something along those lines)

By sanctioning his return, the board of directors has lost all credibility on the issues of diversity, equity, and inclusion.

> We also call upon the Free Software Foundation to revise its bylaws to institute democratic processes for choosing leadership.

Reviewer: ```suggestion
We also call upon the Free Software Foundation to revise its bylaws to institute democratic processes for choosing leadership and increase the transparency of its operations.
```

The myriad software developers who chose to trust the FSF with stewardship of licenses for the code we wrote deserve to have a voice on the leadership of the FSF.
We cannot trust an organization that knowingly makes controversial decisions behind closed doors.

> The FOSS community has led the way for the rest of the wider tech industry, showing that open standards and practices result in better products.

Reviewer: ```suggestion
The FOSS community has led the way for the rest of the wider tech industry, showing that open standards and practices result in better products while respecting users' freedom, privacy, and community.
```

Now we have the opportunity to light a better way for the industry at large by showing that a more welcoming, inclusive industry also creates products that are better for everyone.

> Our community needs to demonstrate a strong sense of accountability at this time, and we trust that the FSF board will do the right thing and accept responsibility for this bad decision.

Reviewer: ```suggestion
Our community needs to demonstrate a strong sense of accountability at this time.
We urge that the FSF board to do the right thing by accepting responsibility for this bad decision and enact specific commitments for change to rebuild trust.
```

title: "A Statement on Richard Stallman and the Free Software Foundation"
authors: Owen Williams
tags: community, diversity, politics
date: 2021-03-26 00:00:00

The contributors to the Mixxx project were dismayed this week to learn that Richard Stallman had been reinstated to the board of directors of the Free Software Foundation.
His departure in 2019 was a consequence of the criticism he faced regarding his inappropriate behavior towards women, his hurtful comments about people with Down's Syndrome and his remarks relating to the Epstein scandal.

> His resignation came as a relief for many who have suffered his bad behavior.

Reviewer 2: ```suggestion
His resignation came as a relief for many who have suffered his actions.
```

Reading this again, we already used "behavior" in the sentence before and it reads a bit strange.

Reviewer 4: I also was not really happy with this sentence but didn't have any alternative at hand. Jan's proposal is perfect and omits the term "bad".

While Mixxx is not officially associated with the Free Software Foundation, there's an inevitable connection in the public's mind between all FOSS projects and the FSF.
Furthermore, Mixxx is and always has been licensed under the terms of the GNU General Public License, version 2 or *later*, and only the Free Software Foundation can publish new versions of the GNU GPL which could be retroactively applied to the Mixxx code.
Given this relationship, the Mixxx project has an interest in ensuring that the Free Software Foundation has inclusive, principled, and wise leadership which can effectively lead the broad free software community to meet present and future challenges.

Therefore, we feel an obligation to add our voices to the chorus of those who have expressed dismay at this turn of events and call for the resignation of Richard Stallman.
The FSF board of directors, by allowing him to return, is giving the appearance of endorsing his conduct and can't remain effective without significant personnel changes and bylaw reforms.
By sanctioning his return, the board of directors has lost all credibility on the issues of diversity, equity, and inclusion.

> We also call upon the Free Software Foundation to revise its bylaws to institute democratic processes for choosing leadership and increase the transparency of its operations.

Reviewer 3: ```suggestion
We acknowledge the Free Software Foundation's efforts to revise its bylaws to establish democratic processes for electing leadership and to increase the transparency of its activities.
```

Reviewer 3: This reflects the current state better. We can't call something in a news addressing the Mixxx user.

We cannot trust an organization that knowingly makes controversial decisions behind closed doors.

> We do acknowledge the Free Software Foundation's efforts to revise its bylaws to establish democratic processes for electing leadership and to increase the transparency of its activities, and the community will be watching closely to verify that these changes are effecive and substantial.

Reviewer: ```suggestion
We do acknowledge the Free Software Foundation's efforts to revise its bylaws to establish democratic processes for electing leadership and to increase the transparency of its activities, and the community will be watching closely to verify that these changes are effective and substantial.
```

The FOSS community has led the way for the rest of the wider tech industry, showing that open standards and practices result in better products while respecting users' freedom, privacy, and community.
Now we have the opportunity to light a better way for the industry at large by showing that a more welcoming, inclusive industry also creates products that are better for everyone.
Our community needs to demonstrate a strong sense of accountability at this time.

> We urge the FSF board to do the right thing by accepting responsibility for this bad decision and enact specific commitments for change to rebuild trust.

Reviewer 3: ```suggestion
We now expect the FSF board to take responsibility for this bad decision and making enact specific commitments for change to rebuild trust.
```
This way it matches to the current development. 

> We urge the

We can urge them in a letter addressed to them and we can tell our users about it.

Reviewer 2: I think "urge" fits better to be honest. And I think we can use it here, as it's basically kind of an open letter.

Reviewer 3: Deepl translates this to: 

Wir fordern den FSF-Vorstand auf, das Richtige zu tun, indem er die Verantwortung für diese Fehlentscheidung übernimmt und konkrete Änderungszusagen macht, um das Vertrauen wiederherzustellen.

But we don't, we write a news blog. 

We could write: We have asked the FSF board ...
If we had done that, but so far we have not. 

Therefore I think it is correct to express our expectations here. In the sense of a last chance. 
Otherwise, we must clearly distance ourselves from the FSF, as Red Head has done. 
A contribution in this sense I would like more, but so it is also OK.

Reviewer 2: Even in German, "urge" ("auffordern") would be correct IMHO. We do it by publishing this article (like an open letter). "We expect" ("wir erwarten") sounds very antagonistic to me. But I don't have a hard opinion to this and leave it to @ywwg to decide.

Reviewer 3: My idea was to acknowledge that they already have announced this step.

Reviewer 2: Ok, maybe something like:
```suggestion
We expect the FSF board to do the right thing and follow up its statements with actions by accepting responsibility for this bad decision and enacting specific commitments for change to rebuild trust.
```

Reviewer 3: Thank you. @ywwg The "expect" was meant like the angry mother expect when telling her son to tidy up. And not like expecting a snail post letter. Don't know if both versions exists in English.

Reviewer: That's good, just change "enact" to "enacting".

Reviewer 2: Done.

> We urge that the FSF board to do the right thing by accepting responsibility for this bad decision and enact specific commitments for change to rebuild trust.

Reviewer: ```suggestion
We urge the FSF board to do the right thing by accepting responsibility for this bad decision and enact specific commitments for change to rebuild trust.
```
